packages/babel-parser/src/parser/location.js

```diff
@@ -67,7 +67,7 @@ export const Errors = Object.freeze({
   IllegalReturn: "'return' outside of function",
   ImportCallArgumentTrailingComma:
     "Trailing comma is disallowed inside import(...) arguments",
-  ImportCallArity: "import() requires exactly one argument",
+  ImportCallArity: "import() requires exactly %0",
```

Comment:
We should also move `exactly` "inside" `%0`, since the error "requires exactly two arguments" when the proposal is enabled is wrong.